CMSgov/mint-app
# [EASI-4145] Change history page (in progress)

# EASI-4145


## Changes and Description

This is a work in progress and will need to be continually worked on as BE unblocks.

- Added routing and flag for change history route
- Added component for rendering a group of changes
- Added component for rendering individual change (list or string)
- Added a util to parse curly bracket string into array (this may change in the future pending BE progress on this)
- Unit tests



## How to test this change'

Some of the translation changes are not complete on the BE.  Enums and strings should work as expected

1. Make some changes in the task list
2. Run `go run ./cmd/dbseed/*go` in the terminal and select the option to translate audits
3. Navigate to `/models/:modelID/change-history`
4. Verify that change history renders according to Figma (not all work is done, just verify lists render and strings render)
5. Verify toggle works

## PR Author Review Checklist



## PR Reviewer Guidelines
- It's best to pull the branch locally and test it, rather than just looking at the code online!
- Check that all code is adequately covered by tests - if it isn't feel free to suggest the addition of tests.
- Always make comments, even if it's minor, or if you don't understand why something was done.
